package/root/usr/local/sbin/resize_rootfs.sh

```diff
@@ -2,6 +2,13 @@
 
 set -xe
 
+if [ "$(id -u)" -ne "0" ]; then
+	echo "This script requires root."
+	exit 1
+fi
+
+apt-get update && apt-get install gdisk parted
```

Comment:
Confirmed missing on 0.2.3 image. 

Should I add them to the below lists, then I can remove the package install and leave just the root check?
https://github.com/ayufan-rock64/linux-rootfs/blob/master/configs/ubuntu/package-lists/additional.list.chroot
https://github.com/ayufan-rock64/linux-rootfs/blob/master/configs/debian/package-lists/additional.list.chroot